core/tauri/src/lib.rs

```diff
@@ -84,12 +81,14 @@ pub use {
 };
 pub use {
   self::app::WindowMenuEvent,
+  self::event::{Event, EventHandler},
```

Comment:
I think only `Event` needs to be exported

Reply:
No we need EventHandler so the user can unlisten to events

Reply:
Ah okay